frontend/app/mstore/userStore.ts

```diff
@@ -54,7 +54,7 @@ export default class UserStore {
         })
     }
 
-    updateKey(key: string, value: any) {
+    updateKey(key: keyof this, value: any) {
```

Comment:
where do we use this for?

Reply:
everywhere to update stuff

I'm removing this methods when I see the opportunity